frontend/admin/.storybook/preview-head.html

```diff
@@ -2,4 +2,15 @@
 storybook decorator. This allows libraries like reach-dialog to work, which
 append elements to the bottom of the body tag enclosure.
 Source: https://github.com/storybookjs/storybook/issues/1672#issuecomment-455621938 -->
-<script>setTimeout(function () { document.body.setAttribute("data-h2", "true"); }, 0);</script>
+<script>
+  document.addEventListener("DOMContentLoaded", function () {
+    document.body.setAttribute("data-h2", "true");
+  });
+</script>
```

Comment:
Seems like a good catch! Out of curiosity, what issue was this raising, or what have you seen previously that led you to add this? :)

Reply:
Sometimes, the `setTimeout` was running before the DOM was loaded which caused no css to be applied. It was a far edge case but did happen.